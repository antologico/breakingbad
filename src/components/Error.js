import React from 'react'
import { FormattedMessage } from 'react-intl'
import { ErrorWrapper, LinkButton } from '../ui'

export default () => <ErrorWrapper>
    <div>
        <FormattedMessage id='error' defaultMessage='Error' />
    </div>
    <LinkButton to='/' data-test='back-button'>
        <span><FormattedMessage defaultMessage='Return' id='return' /></span>
        <span>&#8617;</span> 
    </LinkButton>
</ErrorWrapper>