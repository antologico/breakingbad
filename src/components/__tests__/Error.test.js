import React from 'react'
import { create } from '../../tests/utils'
import Error from '../Error'

jest.mock('react-router-dom', () => ({
  Link: ({children}) => <a id='link'>{children}</a>
}))

describe('Error component', () => {
  test('renders properly', () => {
    const component = create(
        <Error />
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})