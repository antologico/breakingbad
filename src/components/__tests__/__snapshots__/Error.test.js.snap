// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component renders properly 1`] = `
<div
  className="sc-jSMfEi cbmkWi"
>
  <div>
    Se ha producido un error cargando los datos
  </div>
  <a
    id="link"
  >
    <span>
      Volver
    </span>
    <span>
      â†©
    </span>
  </a>
</div>
`;
